Makefile

```diff
@@ -313,7 +313,22 @@ test: ##@test Run tests once in NodeJS
 	yarn shadow-cljs compile test && \
 	node --require ./test-resources/override.js target/test/test.js
 
-#--------------
+component-test-watch: export TARGET := clojure
+component-test: export __COMPONENT_TEST__ := true
+component-test: export BABEL_ENV := test
+component-test-watch: ##@ Watch tests and re-run no changes to cljs files
+	yarn install
+	nodemon --exec 'yarn shadow-cljs compile component-test && jest --config=test/jest/jest.config.js' -e cljs
+
+component-test: export TARGET := clojure
+component-test: export __COMPONENT_TEST__ := true
+component-test: export BABEL_ENV := test
+component-test: ##@test Run tests once in NodeJS
+	# Here we creates the gyp bindings for nodejs
```

Comment:
I am about to make a very lame comment,
brace for impact
.
.
fix grammar to :  "Here we create the gyp bindings for nodejs "